xs/src/libslic3r/Model.cpp

```diff
@@ -20,6 +20,9 @@ Model::Model(const Model &other)
     this->objects.reserve(other.objects.size());
     for (ModelObjectPtrs::const_iterator i = other.objects.begin(); i != other.objects.end(); ++i)
         this->add_object(**i, true);
+
+    // copy metadata
+    this->metadata = other.metadata;
```

Comment:
Where is the initialize for metadata and the part number? I don't see a commit in Model.